NoteFind/View Controllers/NewNoteViewController.m

```diff
@@ -93,14 +92,20 @@ - (void)addBooks:(nonnull NSMutableArray *)selectedBooks {
     NSLog(@"books added: %@", self.tags);
```

Comment:
Try to remove NSLogs from code unless you're debugging

Reply:
resolved